docs/reference/release-notes/7.11.asciidoc

```diff
@@ -153,6 +153,15 @@ Also see <<breaking-changes-7.11,Breaking changes in 7.11>>.
   downgrades are not safe. Elasticsearch 7.11.1 will contain a fix for this
   issue. For more details, see {es-issue}68838[#68838].
 
+* If autoscaling is enabled for machine learning, the administrator of the cluster
+  should increase the cluster setting `xpack.ml.max_open_jobs`. This allows autoscaling
```

Comment:
Is it possible to provide guidance about what it should be set to? e.g. 512 per https://github.com/elastic/elasticsearch/pull/72423?

```suggestion
  should increase the cluster setting `xpack.ml.max_open_jobs` to the maximum value of `512`. This allows autoscaling
```